### ISSUE TITLE
Opening a YARA file fails with "E395: contains argument not accepted here"

### ISSUE THREAD

**Issue by masq** [s3rvac/vim-syntax-yara#1]
Just got a new computer, and I installed this, using pathogen, and I'm greeted with this error when trying to open a YARA file:
```
"rules/test.yar" 40L, 956C
Error detected while processing /Users/$USER/.vim/syntax/yara.vim:
line   38:
E395: contains argument not accepted here
E475: Invalid argument: yaraKeyword all and any ascii at base64 base64wide condition contains entrypoint false filesize
 for fullword global import in include int16 int16be int32 int32be int8 int8be matches meta nocase not of or private ru
le strings them true uint16 uint16be uint32 uint32be uint8 uint8be wide xor
Press ENTER or type command to continue
```

My vim is version 8.1.2292, and I have a pretty vanilla .vimrc (load pathogen, syntastic with some python syntax checkers).

I also tried to do some troubleshooting by removing keywords, and found that if I simply removed the keyword "contains" from the list of keywords, the error went away. Which is... strange. Have you encountered this? Any ideas on what might be causing this?

Thanks, appreciate the syntax highlighting!

**Comment by s3rvac** [comment]
Hi! Thank you for the report and initial investigation, it was very helpful. I believe that you have stumbled upon [this issue](https://stackoverflow.com/questions/55504057/is-there-any-way-to-highlight-default-vim-keyword-without-using-match). Let me fix that :slightly_smiling_face:

**Comment by s3rvac** [comment]
Fixed in commit 403a2b6847e29468dbca972abc3ffb34c9b00b9c.
